lib/header.js

```diff
@@ -32,11 +32,13 @@ vis.js may be distributed under either license.`;
  * @returns {String} banner
  */
 function genHeader(component) {
-	return [
+   return '/*\n' + [
       pkg.name,
       component ? ' - ' + component : '',
       header
-   ].join('');
+   ].join('')
+   .replace(/^(?!\n)/gm, ' * ')
```

Comment:
I would prefer it if you would leave the `header.js` untouched for now. I'm not sure if we want the same copyright-header an every visjs module in the future.

Also one could argue if this empty spaces should be there or not...Let's just not discuss this, yet. I think we have more important things to do :rofl:

Reply:
Okay, I'll move it to rollup.config.js.

Reply:
> Okay, I'll move it to rollup.config.js.

Its not about the `header.js` file, but the header it produces. For now it is the same header overall projects. Maybe this is a good thing!?
Why do you want to change it? Ist eslint unhappy?

Reply:
The problem here is that rollup-plugin-banner messes up source maps. Rollup has native support for banners (headers) but they have to be valid JavaScript comments. This code just turns it into a comment.

Reply:
BTW: The final header looks the same.